src/layouts/EditImage.jsx

```diff
@@ -1,9 +1,8 @@
 import React, { Component } from 'react';
 // import { Link } from "react-router-dom";
 import axios from 'axios';
```

Comment:
There's some strange behavior with the existing PNG and SVG images

Reply:
update: it's just SVG images

Created an issue to track this #8